lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -201,24 +201,40 @@ class UsesHTTP2Audit extends Audit {
   static async audit(artifacts, context) {
     const trace = artifacts.traces[Audit.DEFAULT_PASS];
     const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+    const networkRecords = await NetworkRecords.request(devtoolsLog, context);
+    const resources = UsesHTTP2Audit.determineNonHttp2Resources(networkRecords);
+
+    let displayValue;
+    if (resources.length > 0) {
+      displayValue = str_(UIStrings.displayValue, {itemCount: resources.length});
+    }
+
+    // TODO(FR-COMPAT): Compute actual savings for timespan mode.
+    if (artifacts.GatherContext.gatherMode === 'timespan') {
+      /** @type {LH.Audit.Details.Table['headings']} */
+      const headings = [
```

Comment:
have you tested this in the report UI? I think we move this into the opportunities section based on its `group` in the config and a missing `overallSavingsMs` might be problematic.

Reply:
Yeah, there was an issue with the display value:

![Screen Shot 2021-07-21 at 1 40 53 PM](https://user-images.githubusercontent.com/6752989/126534717-3370004b-c43f-4e0a-9c51-b01bc0f7ef98.png)

I thought it was due to the erroneous audits above (looking into those ATM). Is there a way to split the display group without a separate audit?

Reply:
> Is there a way to split the display group without a separate audit?

Not yet ;) Perhaps quick fix in the report renderer only include in opportunities group if it has `overallSavingsMs` property?

Reply:
this probably points to a deeper problem of `group: 'load-opportunities'` vs `LH.Audit.Details.Opportunity`

e.g. we have loose checks like
https://github.com/GoogleChrome/lighthouse/blob/d62acc3d691c1293c989ed43f2e049f9d5071962/lighthouse-core/test/config/default-config-test.js#L15
but that only tests that the two are equivalent in sample_v2...

Reply:
Current solution is to move audit to diagnostics if details type is not `opportunity`. Not every audit under `group: 'load-opportunities'` needs to return a `LH.Audit.Details.Opportunity`, but the rendered list of opportunities will all be `LH.Audit.Details.Opportunity`.

Reply:
I don't think it's a great thing we have these defined in the config and in audit details, but ideally we wouldn't have to do this incidentally (rather than in a change specifically for changing this).

My main worry is heaping on conditions and it being difficult to follow (why is it an opportunity but not an opportunity?) and difficult to unwind later. Other options:

- could the FR config have `uses-http2` in `group: 'diagnostics'` for timespan?
- does someone want to look into if we can retire `group: 'load-opportunities'` and `group: 'diagnostics'` and just have them be an opportunity if they're details `type: 'opportunity'`, otherwise they're a diagnostic? We'd have to resurrect some form of the "hidden" group (https://github.com/GoogleChrome/lighthouse/pull/11798/commits/e6e439509d270f725729bd05083ee6748fa246f9) for the hidden perf audits, though. It seems like it should work but there may be other things preventing us from doing it.